app/services/clients/stripe/StripeOrganisationDetails.class.js

```diff
@@ -3,14 +3,14 @@
 const Joi = require('joi')
 
 const schema = {
-  name: Joi.string().required(),
+  name: Joi.string().optional(),
```

Comment:
Any reason `name` is optional? This is stripe specific class and name is always required.